src/pages/orders/edit-ticket-page.js

```diff
@@ -197,8 +197,10 @@ class EditTicketPage extends React.Component {
 
         const { refundAmount, refundNotes} = this.state;
         const { refundTicket, entity} = this.props;
-
-        if(parseFloat(refundAmount) > 0 &&  parseFloat(refundAmount) <= parseFloat(entity.final_amount)) {
+        
+        if( parseFloat(refundAmount) > 0 && 
+            parseFloat(refundAmount) <= parseFloat(entity.final_amount) && 
+            parseFloat(refundAmount) <= parseFloat(entity.net_selling_cost)) {
```

Comment:
this is not the proper validation
const maxAllowedAmount2Refund = entity.net_selling_cost - entity.refunded_amount
if(refundAmount > maxAllowedAmount2Refund)  show some inline error informing that